test/e2e/config_arm64.go

```diff
@@ -5,11 +5,11 @@ var (
 	STORAGE_OPTIONS          = "--storage-driver vfs"                                                                                                                                    //nolint:revive,stylecheck
 	ROOTLESS_STORAGE_FS      = "vfs"                                                                                                                                                     //nolint:revive,stylecheck
 	ROOTLESS_STORAGE_OPTIONS = "--storage-driver vfs"                                                                                                                                    //nolint:revive,stylecheck
-	CACHE_IMAGES             = []string{ALPINE, BB, fedoraMinimal, NGINX_IMAGE, REDIS_IMAGE, REGISTRY_IMAGE, INFRA_IMAGE, LABELS_IMAGE, HEALTHCHECK_IMAGE, SYSTEMD_IMAGE, fedoraToolbox} //nolint:revive,stylecheck
+	CACHE_IMAGES             = []string{ALPINE, BB, fedoraMinimal, NGINX_IMAGE, REDIS_IMAGE, REGISTRY_IMAGE, INFRA_IMAGE, CITEST_IMAGE, HEALTHCHECK_IMAGE, SYSTEMD_IMAGE, fedoraToolbox} //nolint:revive,stylecheck
 	NGINX_IMAGE              = "quay.io/lsm5/alpine_nginx-aarch64:latest"                                                                                                                //nolint:revive,stylecheck
 	BB_GLIBC                 = "docker.io/library/busybox:glibc"                                                                                                                         //nolint:revive,stylecheck
 	REGISTRY_IMAGE           = "quay.io/libpod/registry:2.8.2"                                                                                                                           //nolint:revive,stylecheck
-	LABELS_IMAGE             = "quay.io/libpod/alpine_labels:latest"                                                                                                                     //nolint:revive,stylecheck
+	CITEST_IMAGE             = "quay.io/libpod/testimage:20221018"                                                                                                                       //nolint:revive,stylecheck
```

Comment:
Two questions for further consideration.  They don't need to be addressed for this PR.

1) As this variable is referenced in at least two places, is there a place it could be added to and then pulled into here?

2) Just looking at the testimage date.  It's almost a year old, is it time to bump it?

Reply:
Could you rephrase question 1? I can't find a way to understand it. I'm sorry.

Bumping: absent any pressing need, bumping doesn't really gain us anything...